fe/fe-core/src/main/java/com/starrocks/catalog/Catalog.java

```diff
@@ -7486,5 +7484,11 @@ public void setImageJournalId(long imageJournalId) {
         this.imageJournalId = imageJournalId;
     }
 
+    public void clearExpiredLabel() {
```

Comment:
Could you change the name to clearExpiredJobs?

Reply:
done